### PR TITLE
fix: add typography, sficon stories

### PR DESCRIPTION
**Pull request by mayashavin**
# Related issue
N/A

# Scope of work
- Add Typography story
- Update SFIcon story with list of icons.

# Screenshots of visual changes
Screen Shot 2021-01-04 at 11 28 48
Screen Shot 2021-01-04 at 11 28 57
Screen Shot 2021-01-04 at 11 29 08

# Checklist

- [ ] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [ ] Changes documented

### CODE REVIEW

File: packages/vue/src/components/atoms/SfIcon/SfIcon.vue [hunk 1]
```diff
@@ -43,6 +36,16 @@ import icons from "@storefront-ui/shared/icons/icons";
 import { iconColorsValues as SF_COLORS } from "@storefront-ui/shared/variables/colors";
 import { sizesValues as SF_SIZES } from "@storefront-ui/shared/variables/sizes";
 const SF_ICONS = Object.keys(icons);
+
+const fillPathUrl = (index) => `url(#${index})`;
+
+/**
+ * 1. Aria-hidden on by default
```
Comment (justyna-13) [inline packages/vue/src/components/atoms/SfIcon/SfIcon.vue:43]: comments for to do? :)

**Review by justyna-13 — COMMENTED** [review]
Great job @mayashavin :slightly_smiling_face: one fix: changing icon color doesn't work in story

File: packages/vue/src/components/atoms/SfIcon/SfIcon.vue (hunk 1 above)
Comment (mayashavin) [reply packages/vue/src/components/atoms/SfIcon/SfIcon.vue:43]: oops. Yes :) @justyna-13  Removing it now :)

**Comment by mayashavin** [comment]
fixed @justyna-13

**Review by justyna-13 — APPROVED** [review]

**Pull request merged**